package.json

```diff
@@ -29,6 +29,7 @@
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "hardhat coverage --testfiles \"test/unit\" && istanbul check-coverage ./coverage.json --statements 100 --branches 96 --functions 100 --lines 100",
     "test:e2e": "node --experimental-fetch ./node_modules/mocha/bin/mocha.js --exit test/integration",
+    "test:gas": "REPORT_GAS=true npx hardhat test",
```

Comment:
Looks really useful to me. Why do some scripts use `hardhat` directly while others use it via `npx`?

Reply:
Good question!  When a command is run via `npm run` from the "scripts" in package.json we don't need the `npx`.  I'll remove it to be consistent.